books/free-programming-books-ko.md

```diff
@@ -106,6 +102,16 @@
 * [GNOME 배우기](https://sites.google.com/site/gnomekr/home/learning_gnome)
 
 
+### Mathematics
+
+* [기초정수론: 계산과 법연산, 그리고 비밀통신을 강조한](https://wstein.org/ent/) - William Stein
```

Comment:
Could we use the direct link? https://wstein.org/ent/ent_ko.pdf

Reply:
That would be nice! I'll make a direct link.